test/verify/machineslib.py

```diff
@@ -1776,7 +1818,7 @@ def tryCreate(self, dialog):
                     b.wait_in_text("#vm-{0}-disks-vda-device".format(name), "disk")
                 else:
                     b.wait_in_text("#vm-{0}-disks-hda-device".format(name), "disk")
-            elif dialog.storage_pool == 'No Storage' and dialog.sourceType == 'file':
+            elif (dialog.storage_pool == 'No Storage' and dialog.sourceType == 'file') or (dialog.sourceType == 'url' and dialog.start_vm):
```

Comment:
If startVm will not be set the cdrom will be created anyway, no?
No need to change it, I just want to understand it.

Reply:
Actually no.
If startVm is true, cdrom disk is created with iso url parsed into host properties of disk elements.
If startVm is false, cdrom disk is not created, and iso url is stored in metadata of XML.